dom.bs

```diff
@@ -1835,6 +1835,11 @@ service worker.
 <p>The <dfn attribute for=AbortSignal>aborted</dfn> attribute's getter, when invoked, must return
 true if the <a>context object</a>'s [=AbortSignal/aborted flag=] is set, and false otherwise.
 
+<p>The <dfn attribute for=AbortSignal><code>onabort</code></dfn> attribute is an
+<a>event handler IDL attribute</a> for the <dfn for=AbortSignal><code>onabort</code></dfn>
```

Comment:
Bikeshed fails on this because we don't export onabort here. Maybe we should not dfn it?

Reply:
Hmm I swore this built locally.

I'm trying following HTML, which seems to have three definitions in these cases: the IDL attribute, the event handler, and the event.

I think probably the right thing to do is export it, so I'll update it to do that. (And I'll make sure it builds locally.)